.bazelci/presubmit.yml

```diff
@@ -0,0 +1,29 @@
+---
+matrix:
+  bazel_version: ["7.0.0"]
+  platform: ["centos7", "debian10", "macos", "ubuntu2004"]
```

Comment:
This file adds support for Bazel CI, which is used for testing with BCR, I believe.